biz.aQute.bndlib/src/aQute/bnd/classfile/ClassFileRenamer.java

```diff
@@ -0,0 +1,588 @@
+package aQute.bnd.classfile;
```

Comment:
The classfile package is carefully structure for read-only operations. All mutation operation are kept outside of this package. I would prefer to continue this. So we should move all mutation code to a subpackage such as aQute.bnd.classfile.builder or some other subpackage. I want to be clear that the classfile package is not about mutation.